src/app/components/case-reference-search-box/case-reference-search-box.component.ts

```diff
@@ -54,10 +56,12 @@ export class CaseReferenceSearchBoxComponent implements OnInit, OnDestroy, After
   }
 
   public onSubmit(): void {
+    const caseReference = this.formGroup.get(this.CASE_REF_FIELD).value;
```

Comment:
@johnbenjamin-hmcts - just something to be aware of (though I'll pick this up as part of the validation): there's currently nothing that strips spaces or `-` characters from `caseReference`, which could lead to an invalid URL if the user has entered a valid case reference but they've included separator characters.